packages/tinacms/src/toolkit/fields/components/reference/components/popover.tsx

```diff
@@ -18,7 +18,7 @@ const PopoverContent = React.forwardRef<
       align={align}
       sideOffset={sideOffset}
       side="bottom"
-      avoidCollisions={false}
+      avoidCollisions={true}
```

Comment:
by default this is true! https://www.radix-ui.com/primitives/docs/components/popover#content
Let's remove it